toxcore/LAN_discovery.h

```diff
@@ -25,34 +25,27 @@ typedef struct Broadcast_Info Broadcast_Info;
  *
  * @return true on success, false on failure.
  */
-non_null()
-bool lan_discovery_send(const Networking_Core *net, const Broadcast_Info *broadcast, const uint8_t *dht_pk,
-                        uint16_t port);
+bool lan_discovery_send(non_null() const Networking_Core *net, non_null() const Broadcast_Info *broadcast, non_null() const uint8_t *dht_pk, uint16_t port);
 
 /**
  * Discovers broadcast devices and IP addresses.
  */
-non_null()
-Broadcast_Info *lan_discovery_init(const Memory *mem, const Network *ns);
+Broadcast_Info *lan_discovery_init(non_null() const Memory *mem, non_null() const Network *ns);
 
 /**
  * Free all resources associated with the broadcast info.
  */
-nullable(1)
-void lan_discovery_kill(Broadcast_Info *broadcast);
-
+void lan_discovery_kill(nullable() Broadcast_Info *broadcast);
```

Comment:
missing in .c

Reply:
Is this different from other .h/.c file pairs?

Reply:
yeah

Reply:
Well, technically I got there after looking at DHT.h/.c